databricks/koalas/indexes.py

```diff
@@ -958,6 +958,105 @@ def max(self):
 
         return result if len(result) > 1 else result[0]
 
+    def argmax(self):
+        """
+        Return a maximum argument indexer.
+
+        Parameters
+        ----------
+        skipna : bool, default True
+
+        Returns
+        -------
+        maximum argument indexer
+
+        Examples
+        --------
+        >>> kidx = ks.Index([10, 9, 8, 7, 100, 5, 4, 3, 100, 3])
+        >>> kidx
+        Int64Index([10, 9, 8, 7, 100, 5, 4, 3, 100, 3], dtype='int64')
+
+        >>> kidx.argmax()
+        4
+        """
+        max_idx = self.max()
+        sdf = self._internal.sdf
+        sdf = sdf.select(self.to_series()._scol.alias('__index_value__'))
```

Comment:
`self.to_series()._scol` -> `self._scol`

Reply:
Seems also we don't need this temp column. You can just call directly against `self._scol`.

Reply:
it is same reason with https://github.com/databricks/koalas/pull/1165#discussion_r362698852